source/_components/mailbox.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: "Mailbox"
```

Comment:
The new component is just 'Mailbox'  as per discussion with Balloob.  The panel name in the frontend is also 'Mailbox'